cmd/skopeo/inspect.go

```diff
@@ -158,9 +157,14 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 				fmt.Fprintf(stdout, "%s\n", string(out))
 			}
 		} else {
-			row := "{{range . }}" + report.NormalizeFormat(opts.format) + "{{end}}"
+			format := report.NormalizeFormat(opts.format)
```

Comment:
This isn’t necessary; `Formatter.Parse` already calls `NormalizeFormat`. In fact doing so here explicitly interferes with the `\n` logic of https://github.com/containers/podman/pull/15673 , and the `table ` feature.

(in both cases)

Reply:
@mtrmac Updated!